dali/image/tiff.cc

```diff
@@ -90,7 +81,55 @@ Image::ImageDims TiffImage::PeekDims(const uint8_t *encoded_buffer, size_t lengt
   }
 
   // TODO(mszolucha): fill channels count
-  return std::make_tuple(height, width, 0);
+  return {static_cast<int64_t>(height),
+          static_cast<int64_t>(width),
+          0};
+}
+
+}  // namespace legacy_impl
+
+TiffImage::TiffImage(const uint8_t *encoded_buffer, size_t length, dali::DALIImageType image_type) :
+        GenericImage(encoded_buffer, length, image_type) {
+#ifdef DALI_USE_LIBTIFF
```

Comment:
As already suggested:
```suggestion
#if LIBTIFF_ENABLED
```